### PR TITLE
Share article URL

### PR DESCRIPTION
**Pull request by stephanebisson**
Phabricator Link: https://phabricator.wikimedia.org/T234630

### Problem Statement

My friends consume unreliable information.

### Solution

I will share Wikipedia articles about the topics we are discussing on social media so they can learn from a good source.

### Note

This is just a quick and dirty version for fun.

### CODE REVIEW

**Comment by hueitan** [comment]
so nice, it works!
image

**Comment by medied** [comment]
Marking as ready with two bullets: 
- I'm leaving the quickfacts action untouched so we let #351 handle that
- It's possible we might need to update the share icon later

File: src/components/Article.js [hunk 1]
```diff
@@ -223,7 +235,8 @@ const ArticleInner = ({ lang, articleTitle, initialAnchor }) => {
     { name: 'sections', enabled: true, handler: showArticleTocPopup },
     { name: 'quickfacts', enabled: !!article.infobox, handler: showQuickFacts },
     { name: 'gallery', enabled: !!article.media.length, handler: showGallery },
-    { name: 'languages', enabled: article.languageCount, handler: showArticleLanguagePopup }
+    { name: 'languages', enabled: article.languageCount, handler: showArticleLanguagePopup },
+    { name: 'share', enabled: !!window.MozActivity, handler: shareArticleUrl }
```
Comment (stephanebisson) [inline src/components/Article.js:239]: Similar to `shareArticleUrl()` we should propagate `shareEnabled` instead of repeating `!!window.MozActivity`

**Comment by stephanebisson** [comment]
I've merged #351 so if you get up to date with main you should have at most 4 actions.

**Comment by medied** [comment]
Awesome thanks, looking good:

image

**Review by hueitan — APPROVED** [review]
Looks great on device!

**Comment by stephanebisson** [comment]
Merging now to get the "share" string translated asap.

**Pull request merged**